Guide/src/dev_guide/getting_started/linux.md

```diff
@@ -57,7 +57,8 @@ $ sudo apt install \
   binutils              \
   build-essential       \
   gcc-aarch64-linux-gnu \
-  libssl-dev
+  libssl-dev            \
+  pkg-config
```

Comment:
Why is this change included in this PR?

Reply:
This is a drive-by improvement: this is _already_ a requirement (for example to build `disk_crypt`). I did not think it worthwhile to split this into a separate PR, but I am happy to.

Reply:
That's fine. Just wanted to get clarify. :-)

Reply:
I'd prefer to see this in a separate PR. `gh pr create --web` or `gh pr create --fill` makes PRs a low stakes affair, so let's keep separate things separate.